### PR TITLE
Message leaks

### PR DESCRIPTION
**Pull request by tfennelly**
Some load testing tools for the SSE gateway + some fixes found during testing

### CODE REVIEW

File: load-test/package.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+  "name": "sse-gateway-load-test",
+  "version": "1.0.0",
+  "description": "SSE Gateway Sample Load Test",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "mvnbuild": "gulp bundle",
+    "mvntest": "gulp test"
```
Comment (scherler) [inline load-test/package.json]:
```
"start": "mvn hpi:run",
"headless": "node src/main/js/sse-gateway-headless-load.js"
```
Could you add the above?

File: load-test/src/main/java/org/jenkinsci/plugins/ssegateway/load/SSELoadPage.java [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+                    final int numMessages = new Integer(numMessagesParam);
+                    final int intervalMillis = new Integer(intervalMillisParam);
+                    final PubsubBus bus = PubsubBus.getBus();
+
+                    System.out.println("Starting to send messages on channel '" + channelName + "'.");
+                    for (int i = 0; i < numMessages; i++) {
+                        try {
+                            bus.publish(new SimpleMessage()
+                                    .setChannelName(channelName)
+                                    .setEventName("amessage")
```
Comment (scherler) [inline load-test/src/main/java/org/jenkinsci/plugins/ssegateway/load/SSELoadPage.java]: `amessage` typo?

File: load-test/package.json (hunk 1 above)
Comment (tfennelly) [inline load-test/package.json]: @scherler ah good idea ... thanks !!!

File: load-test/src/main/java/org/jenkinsci/plugins/ssegateway/load/SSELoadPage.java (hunk 2 above)
Comment (tfennelly) [inline load-test/src/main/java/org/jenkinsci/plugins/ssegateway/load/SSELoadPage.java]: Not actually ... the load test doesn't actually care about this, so anything is fine in this case.

**Pull request merged**